x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/execution/PlanExecutor.java

```diff
@@ -67,7 +69,8 @@ private EsqlSession newSession(String sessionId, EsqlConfiguration cfg) {
             functionRegistry,
             logicalPlanOptimizer,
             mapper,
-            verifier
+            verifier // ,
+            // breaker
```

Comment:
Leftover?

Reply:
should not have been committed. Removed.